app/views/katello/api/v2/content_views/base.json.rabl

```diff
@@ -15,7 +15,7 @@ end
 
 node :last_published do |content_view|
   unless content_view.versions.empty?
-    content_view.versions.last.created_at
+    content_view.versions.order(:major).last.created_at
```

Comment:
@jlsherrill Thanks for the information. Updated the code as per your suggestion.